playground/AspireEventHub/EventHubs.AppHost/ehstorage.module.bicep

```diff
@@ -1,70 +1,66 @@
-targetScope = 'resourceGroup'
-
-@description('')
+@description('The location for the resource(s) to be deployed.')
 param location string = resourceGroup().location
 
-@description('')
 param principalId string
 
-@description('')
 param principalType string
 
-
-resource storageAccount_59pNEh4K4 'Microsoft.Storage/storageAccounts@2022-09-01' = {
-  name: toLower(take('ehstorage${uniqueString(resourceGroup().id)}', 24))
-  location: location
-  tags: {
-    'aspire-resource-name': 'ehstorage'
-  }
-  sku: {
-    name: 'Standard_GRS'
-  }
-  kind: 'StorageV2'
-  properties: {
-    accessTier: 'Hot'
-    networkAcls: {
-      defaultAction: 'Allow'
+resource ehstorage 'Microsoft.Storage/storageAccounts@2023-01-01' = {
+    name: take('ehstorage${uniqueString(resourceGroup().id)}', 24)
```

Comment:
You will need `toLower()` here.

Reply:
I'm wondering if this is the delta from main or whether this file wasn't regenerated previously so its showing more differences than I'm expecting (e.g. accessTier, allowSharedKeyAccess).

Reply:
For `allowSharedKeyAccess`, the bicep is out of date in main. See

https://github.com/dotnet/aspire/pull/4744


I'm not sure about `accessTier`. I'll look to see if the old CDK defaulted that and if we should be setting it by default.

Reply:
> I'm not sure about accessTier. I'll look to see if the old CDK defaulted that and if we should be setting it by default.

The old CDK was [defaulting this](https://github.com/Azure/azure-sdk-for-net/blob/5283dd0c9ac7713767e1ad7f81018e1ab51f53be/sdk/provisioning/Azure.Provisioning.Storage/src/StorageAccount.cs#L42).

```C#
accessTier: kind == StorageKind.BlobStorage || kind == StorageKind.StorageV2 ? StorageAccountAccessTier.Hot : null))
```

I believe this is the default. When using the new CDK to make a Storage Account, I see:

![image](https://github.com/user-attachments/assets/472bb660-19cb-485f-a030-a9eae0dd2229)


cc @tg-msft

Reply:
OK thats good.

Reply:
If you care about it, I'd specify it.  Those defaults do occasionally change with new service versions.

Reply:
> If you care about it

Do we care about it?

Reply:
Will revert to existing setting for this PR.